src/theme/index.hbs

```diff
@@ -139,14 +139,20 @@
         {{{livereload}}}
 
         {{#if google_analytics}}
+        <!-- Google Analytics Tag -->
         <script>
-        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+            var localAddrs = ["localhost", "127.0.0.1", ""];
 
-        ga('create', '{{google_analytics}}', 'auto');
-        ga('send', 'pageview');
+            // make sure we don't activate google analytics if the developer is 
+            // inspecting the book locally...
+            if (localAddrs.indexOf(document.location.hostname) === -1) {
+                (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){ (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
```

Comment:
it looks like the GA has become slightly more garbled than before ;) Maybe we should pass it through some JS reindenter/beautifier to pretend that its human readable ;)

Otherwise to my limited knowledge, this looks sane

Reply:
Haha, good point. Although the ["official" snippet][o] on their site is just as ugly. I've fixed up that first extra-long line, but I don't think it's worth prettifying the snippet because it'll take up lots of vertical space and isn't exactly meant to be read/edited by humans anyway.

[o]: https://developers.google.com/analytics/devguides/collection/analyticsjs/#the_javascript_tracking_snippet

Reply:
I also checked this on my local environment with a book I've got set up for Google Analytics. Everything *seems* okay.